input/elasticapm/internal/modeldecoder/nullable/nullable.go

```diff
@@ -193,6 +202,31 @@ func (v *Int) Reset() {
 	v.isSet = false
 }
 
+// Int64 stores an int64 value and the
+// information if the value has been set
+type Int64 struct {
```

Comment:
There is a `generator/nullable`, but it seems unrelated to this. And running `make generate` doesn't regenerate this file. So while I don't know what that generator does, I guess it doesn't generate this.

Reply:
Nevermind, the generator seems to only kick in when `nullable.Int64` is used.